SettingsList.py

```diff
@@ -3412,7 +3412,7 @@ class SettingInfos:
         gui_type       = None,
         gui_text       = None,
         shared         = True,
-        choices        = [name for name, item in ItemInfo.items.items() if item.type == 'Item']
+        choices        = [name for name, item in ItemInfo.items.items() if item.type not in ['Drop', 'Event', 'Refill', 'Shop']]
```

Comment:
```suggestion
        choices        = [name for name, item in ItemInfo.items.items() if item.type not in ('Drop', 'Event', 'Refill', 'Shop')]
```

From [the code style guide](https://github.com/OoTRandomizer/OoT-Randomizer/blob/Dev/Notes/code-style.md):

> * Use tuples rather than lists if both can be used interchangeably.

Reply:
committed change